ci/packer/zen/provisioners/inventory/group_vars/pfservers/rc_local.yml

```diff
@@ -2,6 +2,10 @@
 rc_local_service_name: rc-local
 rc_local_commands:
   - |
+    make -C /usr/local/pf conf/ssl/server.pem
```

Comment:
I was all proud of my `cd` commands and my 12 lines script and then @nqb comes in and does it in 3 lines :P